Formula/spatialindex.rb

```diff
@@ -17,4 +17,59 @@ def install
     system "make"
     system "make", "install"
   end
+
+  test do
+    # write out a small program which inserts a fixed box into an rtree
+    # and verifies that it can query it
+    (testpath/"test.cpp").write <<-EOS.undent
```

Comment:
FYI, we indent heredoc content by two spaces. Unfortunately I didn't notice pre-merging, but I'll address it in a follow-up.